src/Promitor.Core.Scraping/Configuration/Model/AzureMetricConfiguration.cs

```diff
@@ -12,14 +16,34 @@ public class AzureMetricConfiguration
         /// </summary>
         public int? Limit { get; set; }
 
+        /// <summary>
+        ///     Information about the dimensions of an Azure Monitor metric
+        /// </summary>
+        public IReadOnlyCollection<MetricDimension> Dimensions { get; set; }
+
         /// <summary>
         ///     Information about the dimension of an Azure Monitor metric
         /// </summary>
+        //[Obsolete("Dimension is deprecated, please use Dimensions instead.")]
```

Comment:
Keeping the obsolete statements would lead to getting resharper warnings again, since I have to refer to the field in other places. Should I just remove them entirely instead of commenting them out?

Reply:
We should keep the obsolete and add a Resharper warning to exclude it, it's fine

Reply:
Okay, this should work now 👍